src/Mgate/PersonneBundle/Entity/Personne.php

```diff
@@ -142,6 +148,29 @@ public function getPoste()
         }
     }
 
+    /** GDPR implementation, erase all personnal data */
+    public function anonymize() {
+        parent::anonymize();
+        $this->prenom = 'a';
```

Comment:
you should leave a space after the `parent` call to make it more visible, also it looks weird to have the opening bracket on the same line for this case

Reply:
Done, thanks.